web/Viewer/Detecting When the Viewer UI Loads

```diff
@@ -0,0 +1,76 @@
+# Detecting When the PSPDFKit UI Loads
+
+## Introduction
+
+When working with PSPDFKit Web SDK, you may need to detect when the UI has fully loaded to determine setting some configurations in your application accordingly.
+
+
+## Proposed Suggestion
+
+While we don't have a specific event for UI loading completion, there are two effective approaches to detect when the UI has loaded:
+
+1. Using [renderPageCallback](https://pspdfkit.com/guides/web/current/rendering/custom-page-rendering/#rendering-pages-with-custom-annotations):
```

Comment:
are the links still valid after the rebranding?

Reply:
@veroo-m 
Updated the link